src/features/defi/components/Overview/Overview.tsx

```diff
@@ -0,0 +1,152 @@
+import {
+  Divider,
+  Flex,
+  ModalCloseButton,
+  ModalHeader,
+  Stack,
+  Stat,
+  StatGroup,
+  StatLabel,
+  Tag,
+} from '@chakra-ui/react'
+import { Asset } from '@shapeshiftoss/types'
+import { Amount } from 'components/Amount/Amount'
+import {
+  AssetDescriptionTeaser,
+  AssetDescriptionTeaserProps,
+} from 'components/AssetDescriptionTeaser'
+import { AssetIcon } from 'components/AssetIcon'
+import { RawText, Text } from 'components/Text'
+
+import { DefiActionButtonProps, DefiActionButtons } from '../DefiActionButtons'
+
+type AssetWithBalance = {
+  balance: string
+  allocationPercentage?: string
+} & Asset
+
+type OverviewProps = {
+  underlyingAssets: AssetWithBalance[]
+  rewardAssets?: AssetWithBalance[]
+  name: string
+  description?: AssetDescriptionTeaserProps
+  asset: Asset
+  balance: string
```

Comment:
can we rename this to `cryptoBalance` or `fiatBalance` or whatever balance it actually is?